ecosystem/typescript/sdk/src/plugins/coin_client.ts

```diff
@@ -54,8 +61,32 @@ export class CoinClient {
       // If this is the first time an account has received the specified coinType,
       // and this is set to false, the transaction would fail.
       createReceiverIfMissing?: boolean;
+      // fungible asset address if you wish to transfer a fungible asset
+      assetAddress?: MaybeHexString;
     },
   ): Promise<string> {
+    if (extraArgs?.assetAddress) {
+      /* eslint-disable no-console */
+      console.warn("to transfer a fungible asset, use `FungibleAssetClient()` class for a better support");
```

Comment:
Is it good practice to use console.log in the SDK/library? This might lead to varying effects depending on where the client runs, e.g. in a Cloud Scheduler job

Reply:
this is just a logging mechanism, I don't think it can lead to anything special other than logs being printed.

Reply:
In certain setups, logs can get scraped and piped into logging analytics system like Datadog automatically. I've seen a few setups like this.

Reply:
even with random `console.warn`? npm packages out there use that method to warn users from present/future changes in the sdk https://github.com/axios/axios/blob/21a5ad34c4a5956d81d338059ac0dd34a19ed094/lib/helpers/deprecatedMethod.js#L17

Reply:
I just wanted to bring this up. Not a blocking comment :)